src/RestrictedPython/transformer.py

```diff
@@ -563,6 +565,33 @@ def visit_FormattedValue(self, node):
         """Allow f-strings without restrictions."""
         return self.node_contents_visit(node)
 
+    def visit_TemplateStr(self, node):
+        """Template strings are allowed by default.
+
+        As Template strings are a very basic template mechanism, that needs
+        additional rendering logic to be useful, they are not blocked by
+        default.
+        Those rendering logic would be affected by RestrictedPython as well.
+        """
+        self.warn(
+            node,
+            'TemplateStr statements are not yet allowed, '
```

Comment:
This Warning is from a draft form of the #303 Pull requests and not valid and should be updated.